education/models.py

```diff
@@ -81,3 +82,24 @@ class Payment(models.Model):
         default=ONLINE_PAYMENT,
         verbose_name="Способ оплаты",
     )
+
+
+class UpdateSubscriptionCourse(models.Model):
+    """Модел на проверку подписки"""
+
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+        related_name="user_subscription",
```

Comment:
`related_name` всегда указываются во множественном числе
```suggestion
        related_name="user_subscriptions",
```
Ниже тоже